app/services/dependency-resolver.js

```diff
@@ -41,6 +41,38 @@ export default Ember.Service.extend({
     });
   },
 
+  resolveAddons: function(addons, dependencies) {
+    let addonPromises = {};
+    Object.keys(addons).forEach((name) => {
```

Comment:
This breaks if there is no addons hash in twiddle.json. You should return addonPromises if addons is null or undefined.